src/Symfony/Component/Translation/Tests/Loader/XliffFileLoaderTest.php

```diff
@@ -143,50 +143,47 @@ public function testTargetAttributesAreStoredCorrectly()
     public function testLoadInvalidResource()
     {
         $this->expectException(InvalidResourceException::class);
-        $loader = new XliffFileLoader();
```

Comment:
Shouldn't you move this line above `expectException()` instead?

Reply:
Yes, I thought it would be much shorter and I would not expect an exception here as it has no arguments, but I can do if you prefer